### DISCUSSION TITLE
Lesson 4 Doubt in Parts of the video about Chainlink are inconsistent with the Chainlink Docs documentation.

### DISCUSSION THREAD

**Discussion by IAm0xAxe** [Cyfrin/foundry-full-course-cu#224] [Q&A] [answered]
Video: https://youtu.be/PhVv8xjexoY?t=475 Use Kovan
Docs:  https://docs.chain.link/data-feeds/using-data-feeds Use Sepolia

Other: And the Kovan test network is no longer linked, and the Chainlink documentation has updates that are mostly inconsistent with the descriptions in the video content. Looking forward to a solution.

**Comment by Shiva-Sai-ssb** [comment, marked as answer]
Hey @IAm0xTime, The Kovan testnet has been deprecated and is no longer supported. It is recommended that you switch to using the Sepolia testnet instead.

**Reply by IAm0xAxe** [reply]
Hey @Shiva-Sai-ssb, Thank you for your answer, I also found out that the later course @PatrickAlphaC is using Sepolia testnet. thank you again.

**Reply by Shiva-Sai-ssb** [reply]
You're welcome.
